### PR TITLE
Add image example to CSS filter function Invert

### LINKED ISSUE #3968: [Edit] CSS [Edit] CSS Filter Functions: invert()
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to a term within CSS Filter Functions. The term entry lives in docs/content/css/concepts/filter-functions/terms/invert/invert.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Added image to demonstrate code block.

Closes #3968 

I changed the code block to be `invert(75%)` instead of `invert(25%)` because it's a better visual example.

Inverting by 25% looks like this:

Screenshot 2023-12-03 140829

While inverting by 75% looks like this:

css-filter-invert-example

### Type of Change

- Editing an existing entry: Add media and update code block

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/filter-functions/terms/invert/invert.md [hunk 1]
```diff
@@ -31,10 +31,14 @@ where a required `<value>` can be one of the following:
...
+Invert an image `75%`:
 
 ```css
 .banner-image {
-  filter: invert(25%);
+  filter: invert(75%);
 }
 ```
+
+The following images demonstrate the invert filter on a background image. The image on the left has no filter, while the image on the right has `filter: invert(75%)` applied.
```
Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/invert/invert.md:42]:
```suggestion
The following output demonstrates the invert filter on a background image. The image on the left has no filter, while the image on the right has a filter `invert(75%)` applied.
```

File: content/css/concepts/filter-functions/terms/invert/invert.md [hunk 2]
```diff
@@ -31,10 +31,14 @@ where a required `<value>` can be one of the following:
...
+
+![CSS invert filter example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filter-invert-example.png)
```
Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/invert/invert.md:44]:
```suggestion
![Compares the original image to the same image with an invert filter applied](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filter-invert-example.png)
```

**Comment by Jamesllllllllll** [comment]
> @Jamesllllllllll I have suggested two changes for you to address. Let me know once you are done making them.
> 
> For the output image `css-filter-invert-example.png`, can you add a higher-quality screenshot? Try pinch-zooming into the screen before taking the screenshot.

I've updated the example image with a better quality version of the same image. The original image would not be corrected by zooming in as zooming does not increase the quality of the image. I used a larger image as the source and has improved the quality.

**Review by ishg-153 — APPROVED** [review]
@Jamesllllllllll Great! This looks ready for a second review.

**Review by avdhoottt — APPROVED** [review]
Ready to merge!!

**Pull request merged**